cypress/support/commands.js

```diff
@@ -29,13 +29,39 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 addMatchImageSnapshotCommand();
 
 Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+  cy.get(`[data-cy^="${selector}"]`);
```

Comment:
The `getByDataCy` command should return the result of `cy.get()` to allow for chaining. Add a `return` statement before `cy.get()`.